spec/unit/hanami/cli/commands/app/generate/action_spec.rb

```diff
@@ -1,299 +1,460 @@
+        # route
+        expect(fs.read("config/routes.rb")).to eq(routes)
 
-      fs.chdir(directory) do
-        view_file = <<~EXPECTED
-          # auto_register: false
-          # frozen_string_literal: true
+        # action
+        expect(fs.directory?("slices/#{slice}/actions/#{controller}")).to be(true)
 
-          require "#{inflector.underscore(slice)}/view"
+        action_file = <<~EXPECTED
+          # frozen_string_literal: true
 
           module #{inflector.classify(slice)}
-            module Views
-              module Books
-                module Bestsellers
-                  module Nonfiction
-                    class #{inflector.classify(action)} < #{inflector.classify(slice)}::View
-                    end
+            module Actions
+              module #{inflector.camelize(controller)}
+                class #{inflector.classify(action)} < #{inflector.classify(slice)}::Action
+                  def handle(*, res)
+                    res.body = self.class.name
                   end
                 end
               end
             end
           end
         EXPECTED
-        expect(fs.read("#{action}.rb")).to eq(view_file)
-      end
-
-      # template
-      expect(fs.directory?(directory = "slices/#{slice}/templates/books/bestsellers/nonfiction")).to be(true)
-
-      fs.chdir(directory) do
-        template_file = <<~EXPECTED
-          <h1>#{inflector.classify(slice)}::Views::Books::Bestsellers::Nonfiction::Index</h1>
-          <h2>slices/#{slice}/templates/books/bestsellers/nonfiction/#{action}.html.erb</h2>
-        EXPECTED
-        expect(fs.read("#{action}.html.erb")).to eq(template_file)
+        expect(fs.read("slices/#{slice}/actions/#{controller}/#{action}.rb")).to eq(action_file)
+
+        # # view
+        # expect(fs.directory?("slices/#{slice}/views/#{controller}")).to be(true)
+        #
+        # view_file = <<~EXPECTED
+        #   # auto_register: false
+        #   # frozen_string_literal: true
+        #
+        #   require "#{inflector.underscore(slice)}/view"
+        #
+        #   module #{inflector.classify(slice)}
+        #     module Views
+        #       module #{inflector.camelize(controller)}
+        #         class #{inflector.classify(action)} < #{inflector.classify(slice)}::View
+        #         end
+        #       end
+        #     end
+        #   end
+        # EXPECTED
+        # expect(fs.read("slices/#{slice}/views/#{controller}/#{action}.rb")).to eq(view_file)
+
+        # template
+        # expect(fs.directory?("slices/#{slice}/templates/#{controller}")).to be(true)
+        #
+        # template_file = <<~EXPECTED
+        #   <h1>#{inflector.classify(slice)}::Views::#{inflector.camelize(controller)}::#{inflector.classify(action)}</h1>
+        #   <h2>slices/#{slice}/templates/#{controller}/#{action}.html.erb</h2>
+        # EXPECTED
+        # expect(fs.read("slices/#{slice}/templates/#{controller}/#{action}.html.erb")).to eq(template_file)
```

Comment:
Is it really needed to leave this as commented out vs just removing it?

Reply:
@solnic When we'll have views (hopefully soon), it will be a matter of uncommenting those lines.